Extensions/TileMap/tilemapruntimeobject-pixi-renderer.ts

```diff
@@ -146,11 +148,11 @@ namespace gdjs {
     }
 
     getWidth(): float {
-      return this._pixiObject.width;
+      return this._pixiTileMapData ? this._pixiTileMapData.tilemapWidth : this._pixiObject.width;
     }
 
     getHeight(): float {
-      return this._pixiObject.height;
+      return this._pixiTileMapData ? this._pixiTileMapData.tilemapHeight: this._pixiObject.height;
```

Comment:
@4ian note that I am also touching the runtime renderer and indeed without this line - and without a custom size - one of the tilemaps will fail to show up when you playtest

Reply:
I suspect its caused by using this._pixiObject.scale to set scale. Pixi-tilemap is likely overriding it with crazy values

Reply:
Ok I'll take a look! You might want to always use the tilemapWidth/tilemapHeight divided by the scale.x/scale.y

Reply:
Yes, it's indeed I think the problem. 
In the IDE, you need to implement the **default** width/height (which is equal to `this._pixiTileMapData.tilemapWidth` or `this._pixiTileMapData.tilemapHeight`).

Here you must return the actual width or height of the obejct, so:

`this._pixiTileMapData ? this._pixiTileMapData.tilemapHeight * this._pixiObject.scale.y : 0`